pitr/ddl.go

```diff
@@ -481,3 +558,18 @@ func (d *DDLHandle) insertMapKeyFromDB(newKey, oldKey string) error {
 func skipJob(job *model.Job) bool {
 	return !job.IsSynced() && !job.IsDone()
 }
+
+func (d *DDLHandle) ShiftMetaToTiDB() error {
+	var DBInfos []*model.DBInfo
+	d.lastDBInfoMap.Range(func (key, value interface{}) bool {
+		DBInfos = append(DBInfos, value.(*model.DBInfo))
+		return true
+	})
+	return d.tidbServer.SetDBInfoMeta(DBInfos)
+}
+
+func (d *DDLHandle) SetServerHistoryAccelerate(server *tidblite.TiDBServer, jobs []*model.Job, ac bool) {
```

Comment:
useless function

Reply:
DDL pitr test (in new package) wanna use this.